<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1015</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QFrame {
	background-color: #25292B;
}

#centralWidget, QStackedWidget, QStackedWidget &gt; QWidget {
	background: transparent;
}

QLabel {
	color: #F8F8F8;
	background: transparent;
}

QFrame[titleFrame=true] {
	background-color: #5F5F5F;
}

QWidget[fontSize=&quot;14&quot;] {
	font-size: 14pt;
}

QWidget[fontSize=&quot;12&quot;] {
	font-size: 12pt;
}

QGroupBox {
	color: #F8F8F8;
	border: none;
	margin: 15px;
}

QWidget[inList=true] {
	font-size: 12pt;
	min-height: 60px;
	max-height: 100px;
}


/* PushButton */

QPushButton {
	background-color: #25292B;
	border: 2px solid #F8F8F8;
	color: #F8F8F8;
}

QPushButton:hover {
	background-color: #F8F8F8;
	color: #25292B;
}

QPushButton:pressed {
	background-color: #66ff66;
	color: #25292B;
	border: 2px solid #00F0F0;
}


/* RadioButton */

QRadioButton {
	background-color: #25292B;
	border: 2px solid #F8F8F8;
	color: #F8F8F8;
}

QRadioButton:hover {
	background-color: #F8F8F8;
	color: #25292B;
}

QRadioButton:checked {
	background-color: #66ff66;
	color: #25292B;
	border: 2px solid #00F0F0;
}

QRadioButton::indicator {
	width: 0px;
	height: 0px;
	color: #F8F8F8;
}


/* CheckBox */

QCheckBox {
	background-color: #25292B;
	border: 2px solid #F8F8F8;
	color: #F8F8F8;
}

QCheckBox:hover {
	background-color: #F8F8F8;
	color: #25292B;
}

QCheckBox:checked {
	background-color: #66ff66;
	color: #25292B;
	border: 2px solid #00F0F0;
}

QCheckBox::indicator {
	width: 0px;
	height: 0px;
}

/* Slider */
QSlider::groove:horizontal {
    border: 1px solid #999999;
    height: 35px; /* the groove expands to the size of the slider by default. by giving it a height, it has a fixed size */
    background: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #B1B1B1, stop:1 #c4c4c4);
    margin: 2px 0;
}

QSlider::handle:horizontal {
    background: qlineargradient(x1:0, y1:0, x2:1, y2:1, stop:0 #b4b4b4, stop:1 #66ff66);
    border: 1px solid #5c5c5c;
    width: 35px;
    margin: -2px 0; /* handle is placed by default on the contents rect of the groove. Expand outside the groove */
    border-radius: 0px;
}

/* Page Selectors */

QRadioButton[pageSelector=true] {
	background-color: #25292B;
	max-width: 90px;
	max-height: 90px;
	border-radius: 40px;
	border: none;
	padding: 10px;
}

QRadioButton[pageSelector=true]:checked {
	background-color: #F8F8F8;
	max-width: 90px;
	max-height: 90px;
	border-radius: 40px;
	border: none;
	padding: 10px;
}

#viewRadioButton {
	image: url(:Qt/menu/icons/view-white.svg);
}

#viewRadioButton:checked {
	image: url(:Qt/menu/icons/view-black.svg);
}

#toolsRadioButton {
	image: url(:Qt/menu/icons/tools-white.svg);
}

#toolsRadioButton:checked {
	image: url(:Qt/menu/icons/tools-black.svg);
}

#groupsRadioButton {
	image: url(:Qt/menu/icons/groups-white.svg);
}

#groupsRadioButton:checked {
	image: url(:Qt/menu/icons/groups-black.svg);
}

#pauseRadioButton {
	image: url(:Qt/menu/icons/pause-white.svg);
}

#pauseRadioButton:checked {
	image: url(:Qt/menu/icons/pause-black.svg);
}

#playRadioButton {
	image: url(:Qt/menu/icons/play-white.svg);
}

#playRadioButton:checked {
	image: url(:Qt/menu/icons/play-black.svg);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>0</y>
      <width>751</width>
      <height>751</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="timeViewPage">
     <widget class="QFrame" name="frame_5">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>-1</y>
        <width>751</width>
        <height>111</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <property name="titleFrame" stdset="0">
       <bool>true</bool>
      </property>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>-1</y>
         <width>751</width>
         <height>111</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:28pt; font-weight:600;&quot;&gt;Mars Options&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>130</y>
        <width>751</width>
        <height>621</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="spacing">
        <number>24</number>
       </property>
       <item>
        <widget class="QWidget" name="widget" native="true">
         <widget class="QWidget" name="layoutWidget_2">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>751</width>
            <height>241</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QSplitter" name="splitter">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <widget class="QLabel" name="label">
              <property name="font">
               <font>
                <pointsize>14</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Level Of Detail: </string>
              </property>
             </widget>
             <widget class="QLCDNumber" name="lcdNumber"/>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QFrame" name="frame_3">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>260</y>
            <width>749</width>
            <height>73</height>
           </rect>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <widget class="QSlider" name="horizontalSlider">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>0</y>
             <width>731</width>
             <height>71</height>
            </rect>
           </property>
           <property name="minimum">
            <number>-100</number>
           </property>
           <property name="maximum">
            <number>100</number>
           </property>
           <property name="pageStep">
            <number>5</number>
           </property>
           <property name="value">
            <number>0</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="tickPosition">
            <enum>QSlider::NoTicks</enum>
           </property>
          </widget>
         </widget>
         <widget class="QWidget" name="verticalLayoutWidget">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>350</y>
            <width>751</width>
            <height>271</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <property name="spacing">
            <number>9</number>
           </property>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QRadioButton" name="showParticlesRadioButton">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="font">
                <font>
                 <pointsize>14</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Show Particles</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="showShortTrailsRadioButton">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="font">
                <font>
                 <pointsize>14</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Show Short Trails</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="showFullTrailsRadioButton">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="font">
                <font>
                 <pointsize>14</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Show Full Trails</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QCheckBox" name="MarsCheckBox">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="font">
                <font>
                 <pointsize>14</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Show Mars</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="MarsAxesCheckBox">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="font">
                <font>
                 <pointsize>14</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Show Axes</string>
               </property>
               <property name="checked">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="showSkyboxCheckBox">
               <property name="enabled">
                <bool>true</bool>
               </property>
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="font">
                <font>
                 <pointsize>14</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Show Skybox</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="toolsPage">
     <widget class="QFrame" name="frame_4">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>130</y>
        <width>751</width>
        <height>181</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QGroupBox" name="groupBox_7">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>10</y>
         <width>711</width>
         <height>161</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="title">
        <string>Tools</string>
       </property>
       <widget class="QWidget" name="horizontalLayoutWidget_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>55</y>
          <width>711</width>
          <height>81</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>9</number>
         </property>
         <item>
          <widget class="QRadioButton" name="drawButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>&amp;Draw</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="measureButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Meas&amp;ure</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="sphereButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Se&amp;lection Sphere</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="resetButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Reset</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
     <widget class="QFrame" name="frame_6">
      <property name="geometry">
       <rect>
        <x>-1</x>
        <y>0</y>
        <width>751</width>
        <height>111</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <property name="titleFrame" stdset="0">
       <bool>true</bool>
      </property>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>-2</x>
         <y>-1</y>
         <width>751</width>
         <height>111</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:28pt; font-weight:600;&quot;&gt;Tools &amp;amp; Selection&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </widget>
     <widget class="QFrame" name="frame_7">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>330</y>
        <width>751</width>
        <height>421</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QGroupBox" name="groupBox_8">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>10</y>
         <width>711</width>
         <height>401</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="title">
        <string>Selections</string>
       </property>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>410</x>
          <y>50</y>
          <width>301</width>
          <height>341</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <property name="spacing">
          <number>20</number>
         </property>
         <item>
          <widget class="QPushButton" name="saveSelectionsButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Save Selections</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="showAllButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Show All</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="hideAllButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Hide All</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QScrollArea" name="scrollArea">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>50</y>
          <width>381</width>
          <height>341</height>
         </rect>
        </property>
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="contentswidget">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>379</width>
           <height>339</height>
          </rect>
         </property>
         <widget class="QWidget" name="verticalLayoutWidget_5">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>381</width>
            <height>341</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="selectionArea"/>
         </widget>
        </widget>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="systemPage">
     <widget class="QWidget" name="gridLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>110</y>
        <width>731</width>
        <height>541</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <layout class="QGridLayout" name="gridLayout_7">
       <item row="0" column="0">
        <widget class="QPushButton" name="loadButton">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Load</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="preferencesButton">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Preferences</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="saveButton">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Save</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="aboutButton">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>About</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>-1</y>
      <width>175</width>
      <height>751</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="_2">
     <item>
      <widget class="QRadioButton" name="viewRadioButton">
       <property name="text">
        <string/>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
       <property name="pageSelector" stdset="0">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="toolsRadioButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pageSelector" stdset="0">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionLoad">
   <property name="text">
    <string>Load</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionPref_s">
   <property name="text">
    <string>Pref's</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionDraw">
   <property name="text">
    <string>Draw</string>
   </property>
  </action>
  <action name="actionMeasure">
   <property name="text">
    <string>Measure</string>
   </property>
  </action>
  <action name="actionWaypoint">
   <property name="text">
    <string>Waypoint</string>
   </property>
  </action>
  <action name="actionNote">
   <property name="text">
    <string>Note</string>
   </property>
  </action>
  <action name="actionSphere">
   <property name="text">
    <string>Sphere</string>
   </property>
  </action>
  <action name="actionBox">
   <property name="text">
    <string>Box</string>
   </property>
  </action>
  <action name="actionSaveSourceIDs">
   <property name="text">
    <string>Save Source_ID's</string>
   </property>
  </action>
  <action name="actionReset">
   <property name="text">
    <string>Reset</string>
   </property>
  </action>
  <action name="actionPlace1">
   <property name="text">
    <string>Place1</string>
   </property>
  </action>
  <action name="actionPlace2">
   <property name="text">
    <string>Place2</string>
   </property>
  </action>
  <action name="actionPlace3">
   <property name="text">
    <string>Place3</string>
   </property>
  </action>
  <action name="actionPlace4">
   <property name="text">
    <string>Place4</string>
   </property>
  </action>
  <action name="actionShow_All">
   <property name="text">
    <string>Show All</string>
   </property>
  </action>
  <action name="actionHide_All">
   <property name="text">
    <string>Hide All</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../../resources.qrc"/>
 </resources>
 <connections/>
</ui>
